src/networks.d.ts

```diff
@@ -0,0 +1,12 @@
+/// <reference types="node" />
+import { Network as BitcoinJSNetwork } from 'ecpair/src/networks';
+export declare type GenesisBlockHash = Buffer;
+export interface Network extends BitcoinJSNetwork {
+    blech32: string;
+    assetHash: string;
+    confidentialPrefix: number;
+    genesisBlockHash: GenesisBlockHash;
```

Comment:
I would make it optional, to no break all existing Network definition

Reply:
network definition are often (always?) fetched from liquidjs isn't it? the new definitions includes the genesis block so it shouldn't break if the app updates liquidjs? Moreover, genesis hash are constants so even if someone defines a new regtest network, he could use the one exported via `regtest` (so easy to update)